web/src/components/UploadLicenseFile.tsx

```diff
@@ -0,0 +1,683 @@
+import React, { useEffect, useReducer } from "react";
+import { useHistory } from "react-router";
+import { Link } from "react-router-dom";
+import { KotsPageTitle } from "@components/Head";
+// TODO: upgrade this dependency
+// @ts-ignore
+import Dropzone from "react-dropzone";
```

Comment:
I tried adding the `@types` but I started getting a bunch of errors. I think we might be using this component the wrong way.

Reply:
are you referring to the `js-yaml` package?

Reply:
I was able to add the types for that one and get it to work but `Dropzone` is like ten versions behind.